terraform/warehouse/main.tf

```diff
@@ -18,18 +18,20 @@ locals {
 
 
 resource "fastly_service_vcl" "pypi" {
-  name     = "PyPI"
+  name     = var.name
   # Set to false for spicy changes
   activate = true
```

Comment:
```suggestion
  activate = false
```

Reply:
Let's disable auto activation of Fastly services before merge. That way we can double check the diff for the production service to make sure it doesn't break!